README.md

```diff
@@ -1,5 +1,232 @@
-# iota-sdk
+# IOTA SDK Library
 
 [![Coverage Status](https://coveralls.io/repos/github/iotaledger/iota-sdk/badge.svg?branch=develop)](https://coveralls.io/github/iotaledger/iota-sdk?branch=develop)
 
-WIP
+The IOTA SDK is a Rust-based project that provides a convenient and efficient way to interact with Shimmer nodes in the
+Shimmer network. It consists of two main modules: `wallet` and `client`.
+
+The `wallet` module is stateful, with a standardized interface for developers to build applications involving value
+transactions. It uses high-level functions that simplify everyday operations. It can optionally interact
+with [IOTA Stronghold](https://github.com/iotaledger/stronghold.rs/) for seed handling and storage and state backup.
```

Comment:
seed handling, storage and state backup*

Reply:
```suggestion
with [IOTA Stronghold](https://github.com/iotaledger/stronghold.rs/) for seed handling, storage, and state backup.
```